### ISSUE TITLE
"Check for Minor Updates" updates dependencies with --save-exact flag

### ISSUE THREAD

**Issue by danis039** [ionic-team/vscode-ionic#163]
When using the "Check for Minor Updates" functionality and updating dependencies, the extension updates each package with the `--save-exact" flag, which in turn updates package.json itself. Would it be possible to remove this behavior or at least make it configurable in the extension's settings? This would allow developers who use version ranges such as "^1.0.0" (as opposed to "1.0.0") to keep their package.json intact.

Many thanks and keep up the great work!

**Comment by dtarnawsky** [comment]
No. If you want to use version ranges then run `npm update` instead. The extension is designed to pin your dependency versions.
